README.md

```diff
@@ -66,11 +66,33 @@ To remove the Serverless app from Twilio, run the following command:
 
 ## Local Development
 
-In order to develop this app on your local machine, you will first need to deploy all needed endpoints to Twilio Serverless. To do this, complete the steps in the "Deploy the App" section above.
+### Running a local token server
 
-Once the endpoints are deployed, add the app's URL as the `PROXY_URL` in the `.env` file. Then you can start a local development server by running the following command:
+This application requires an access token to run the [Preflight](src/components/AppStateProvider/usePreflightTest/usePreflightTest.ts) and [Bitrate](src/components/AppStateProvider/useBitrateTest/useBitrateTest.ts) tests. The included local token [server](server/index.ts) provides the application with access tokens and TURN credentials. This token server can be used to run the app locally, and it is the server that is used when this app is run in development mode with `npm start`. Perform the following steps to setup the local token server:
 
-    $ npm run start
+- If you haven't done so already, create an account in the [Twilio Console](https://www.twilio.com/console).
+- Click on 'Settings' and take note of your Account SID.
```

Comment:
```suggestion
- If you haven't done so already, create an account in the [Twilio Console](https://www.twilio.com/console) and take note of your Account SID.
```
I don't think there's a 'settings' button to click on. The account sid should just be right there when you log into the console.

Reply:
Oops copied this over from the React app, we will have to change it there too at some point 😄